src/brod_group_coordinator.erl

```diff
@@ -682,19 +682,32 @@ do_commit_offsets_(#state{ groupId                  = GroupId
       },
   Rsp = send_sync(SockPid, Req),
   #kpro_OffsetCommitResponse{oCRspTopic_L = Topics} = Rsp,
-  lists:foreach(
-    fun(#kpro_OCRspTopic{topicName = Topic, oCRspPartition_L = Partitions}) ->
-      lists:foreach(
-        fun(#kpro_OCRspPartition{partition = Partition, errorCode = EC}) ->
-          kpro_ErrorCode:is_error(EC) andalso
-            begin
+  ErrorSet = lists:foldl(
+    fun(#kpro_OCRspTopic{topicName = Topic, oCRspPartition_L = Partitions},
+        Acc1) ->
+      lists:foldl(
+        fun(#kpro_OCRspPartition{partition = Partition, errorCode = EC},
+            Acc2) ->
+          case kpro_ErrorCode:is_error(EC) of
+            true -> 
               log(State, error,
-                  "failed to commit offset for topic=~s, partition=~p\n"
-                  "~p:~s", [Topic, Partition, EC, kpro_ErrorCode:desc(EC)]),
-              erlang:error(EC)
-            end
-        end, Partitions)
-    end, Topics),
+                "failed to commit offset for topic=~s, partition=~p\n"
```

Comment:
to keep it simple: send all error codes to crash log (as in my example code).
I'm not even sure if kafka would in practical return different error codes.